### PR TITLE
Android - styling of consent 2 page

### PR DESCRIPTION
**Pull request by fsnNC**

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml [hunk 1]
```diff
@@ -45,95 +45,102 @@
...
+                android:id="@+id/information_consent_notification_text_textview"
                 style="@style/PrimaryTextRegular"/>
-        </RelativeLayout>
+			<TextView
+				android:text="Be aware text"
+				android:layout_width="wrap_content"
+				android:layout_height="wrap_content"
+				android:paddingTop="48dp"
+                android:paddingLeft="24dp"
+				android:paddingRight="24dp"
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml:105]: are those tabs wrong here or is it just github that does not show it correctly?

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml [hunk 2]
```diff
@@ -45,95 +45,102 @@
...
+                android:textSize="16sp" 
+                android:layout_below="@id/information_consent_notification_text_textview"
+                android:id="@+id/information_consent_beAware_text_textview"
+                style="@style/PrimaryTextBold"/>
+            <TextView
+				android:text="Notification text"
+				android:layout_width="wrap_content"
+				android:layout_height="wrap_content"
+				android:paddingTop="24dp"
+                android:paddingLeft="24dp"
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml:115]: the same as above

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml (hunk 1 above)
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml:105]: Run code autoformatting on the file to be sure everything is ok

Comment (fsnNC) [reply NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml:105]: Hmmm.. weird. That must be Github doing something.
image

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml (hunk 2 above)
Comment (fsnNC) [reply NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml:115]: Hmmm.. weird. That must be Github doing something.
image

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml (hunk 1 above)
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml:105]: some of the intends are spaces and some of them are tabs. so run auto-formatting as i have suggested. it should correct those white spaces.

Comment (fsnNC) [reply NDB.Covid19/NDB.Covid19.Droid/Resources/layout/information_and_consent.xml:104]: should be corrected now

**Review by nc-kano — APPROVED** [review]

**Pull request merged**